.changeset/rude-flowers-mate.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/cms-admin": minor
+"@comet/cms-api": minor
+---
+
+Add AzureAiTranslator to be availabe in all Comet Projects
+To enable Ai Translation add enabled prop to the Provider and add the following env Variables:
+
+-   AZURE_AI_TRANSLATOR_ENDPOINT
+-   AZURE_AI_TRANSLATOR_KEY
+-   AZURE_AI_TRANSLATOR_REGION,
```

Comment:
This is necessary only in Demo. The changeset should show how to add the module to the AppModule

Reply:
e07c9e3ba4b8e66d7003fa41718b98ef2e3bbed6